src/theme.js

```diff
@@ -1,4 +1,62 @@
-export default {
+// export default {
```

Comment:
If it's all commented out, it can go, can't it?

Reply:
True. Originally I had another theme but I guess they are now nicely coalesing. I will remove the comment.